pkg/git/git.go

```diff
@@ -0,0 +1,74 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package git
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing/transport"
+	"github.com/go-git/go-git/v5/storage/memory"
+	"github.com/shipwright-io/build/pkg/ctxlog"
+	"regexp"
+
+	gitpackage "github.com/go-git/go-git/v5"
```

Comment:
how does the alias "gitpackage" have anything to do with this being the go-git package? How about something like gogit or gogitv5?

Reply:
+1